.travis.yml

```diff
@@ -58,6 +58,7 @@ after_failure:
 env:
   global:
     - SEED=1
+    - LOG_FAILURE_ON_STDOUT=1
```

Comment:
What does this do that the call to `tests/scripts/travis-log-failure.sh` doesn't do?

Reply:
Oh, I wasn't aware of that script.